test/subcommand_flags.cc

```diff
@@ -0,0 +1,314 @@
+#include <array>
+
+#include "gtest/gtest.h"
+#include "stout/flags.h"
+#include "test/test.pb.h"
+
+TEST(FlagsTest, MissingSubcommandExtension) {
+  EXPECT_DEATH(
+      []() {
+        test::SubcommandFlagsWithoutExtension flag;
+        auto builder = stout::flags::Parser::Builder(&flag);
+        builder.Build();
+      }(),
+      "Every field of the 'oneof subcommand' must have"
+      " .stout.v1.subcommand. extension");
```

Comment:
```suggestion
      " stout.v1.subcommand extension");
```

Here and everywhere else!